zscaler_api_talkers/zia_talker/zia_talker.py

```diff
@@ -285,60 +352,66 @@ def list_url_categories(self, custom=False):
             url = "/urlCategories"
         # return self._obtain_all(url)
         response = self.hp_http.get_call(
-            url, cookies=self.cookies, error_handling=True, headers=self.headers
+            url,
+            cookies=self.cookies,
+            error_handling=True,
+            headers=self.headers,
         )
+
         return response.json()
 
-    def list_url_categories_lite(self):
+    def list_url_categories_lite(self) -> json:
         """
         Gets a lightweight key-value list of all or custom URL categories.
+
+        :return: (json)
         """
         url = "/urlCategories/lite"
         response = self.hp_http.get_call(
-            url, cookies=self.cookies, error_handling=True, headers=self.headers
+            url,
+            cookies=self.cookies,
+            error_handling=True,
+            headers=self.headers,
         )
+
         return response.json()
 
     def add_url_categories(
         self,
-        name,
-        superCategory,
-        type="URL_CATEGORY",
-        urls=None,
-        dbCategorizedUrls=None,
-        keywordsRetainingParentCategory=[],
-        keywords=[],
-        customCategory=False,
-        ipRanges=[],
-        ipRangesRetainingParentCategory=[],
-    ):
+        name: str,
+        superCategory: str,
+        type: list = ["URL_CATEGORY"],  # FIXME: Shadows built-in name.
```

Comment:
This is a good point.  expected payload and name came directly from https://help.zscaler.com/zia/url-categories#/urlCategories-post